crates/modelardb_common/src/metadata/table_metadata_manager.rs

```diff
@@ -1598,7 +1655,7 @@ mod tests {
         let (_temp_dir, metadata_manager) = create_metadata_manager_and_save_model_table().await;
 
         let result = metadata_manager
-            .mapping_from_hash_to_tags("missing_model_table", &["tag"])
+            .mapping_from_hash_to_tags("missing_table", &["tag"])
```

Comment:
Removed it because we use "missing_table" consistently when testing methods with a table that does not exist, no matter if it is a normal table or a model table.